src/generator/backend/fold.js

```diff
@@ -0,0 +1,224 @@
+'use strict';
+
+
+var exports = module.exports = {};
+
+const fs = require('fs-extra');
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["fs" is defined but never used](https://www.codacy.com/app/triplez/open-event-webapp/file/2737479688/issues/source?bid=3233857&fileBranchId=3352811#l6)